src/modules/endpoint/sagas/loadEvents.js

```diff
@@ -9,6 +9,7 @@ import determineApiUrl from '../determineApiUrl'
 function * loadEvents (
   city: string,
   language: string,
+  eventsEnabled: boolean,
```

Comment:
Maybe we should switch to an object as param at some point.

Reply:
Yeah, but I don't want to bloat this PR.